profiles/seccomp/default.json

```diff
@@ -188,6 +188,7 @@
 				"lstat64",
 				"madvise",
 				"memfd_create",
+				"membarrier",
```

Comment:
@Julio-Guerra could you keep this sorted alphabetically, and move this one line up (before `memfd_create`)? 

CI is failing because this file is generated. To update this profile, you need to add the syscall to https://github.com/moby/moby/blob/master/profiles/seccomp/seccomp_default.go (and regenerate the JSON or edit it manually, I expect regenerating is not needed)

Reply:
Sorry about that and thanks for the details.
It should be good now.